tests/examples/example1.bats

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bats
+
+@test "addition using dc" {
```

Comment:
This is a good "proof" that it works. Let's replace though with something we would actually use.

Reply:
Also, wire it up with codefresh

Reply:
I removed `tests` for now since we are still deciding where all the tests go